DropboxMoveFile/index.js

```diff
@@ -0,0 +1,21 @@
+const main = async (inputs, auths, event) => {
+  const fetch = require('isomorphic-fetch');
+  const { Dropbox } = require('dropbox');
+  const dbx = new Dropbox({
+    accessToken: auths.DROPBOX.ACCESS_TOKEN,
+    fetch
+  });
+
+  try {
+    const resp = await dbx.filesMoveV2({
+      fromPath: inputs.fromPath,
+      toPath: inputs.toPath
+    })
+    return resp
+  } catch (err) {

```

Comment:
No need to use try catch, Dcoder system catches the error for you and let users know. Remove the try catch here let system decide what to do in case of error.